plutil: add page (#4343)
tldr-pages/tldr: pages/osx/plutil.md

# plutil

> > View, convert, validate, or edit property list ("plist") files.

Reviewer: Note: I found some man pages like https://www.theiphonewiki.com/wiki/Plutil#Apple.27s_plutil but is not synced with the mac options.

> - Dump the contents of plist files in human-readable format:

Reviewer: Maybe this description is more readable
```suggestion
- Display the contents of one or more space-separated plist files in human-readable format:
```

> `plutil -p {{file1.plist}} {{file2.plist}}`

Reviewer: We use this format:
```suggestion
`plutil -p {{path/to/file1.plist}} {{path/to/file2.plist}}`
```

> `plutil -p {{path/to/file1.plist}} {{path/to/file2.plist}}`

Reviewer 2: ```suggestion
`plutil -p {{file1.plist file2.plist ...}}`
```

> - Convert plist files to XML format, overwriting the original files in-place:

Reviewer: ```suggestion
- Convert one or more space-separated plist files to XML format, overwriting the original files in-place:
```

> `plutil -convert xml1 {{path/to/file1.plist}} {{path/to/file2.plist}}`

Reviewer 2: ```suggestion
`plutil -convert xml1 {{file1.plist file2.plist ...}}`
```

> - Convert plist files to binary format, overwriting the original files in-place:

Reviewer: ```suggestion
- Convert one or more space-separated plist files to binary format, overwriting the original files in-place:
```

`plutil -convert binary1 {{file1.plist}} {{file2.plist}}`

> - Convert a single plist file to a different format, writing to a new file:

Reviewer: ```suggestion
- Convert a plist file to a different format, writing to a new file:
```

> `plutil -convert binary1 {{path/to/file1.plist}} {{path/to/file2.plist}}`

Reviewer 2: ```suggestion
`plutil -convert binary1 {{file1.plist file2.plist ...}}`
```

> `plutil -convert {{xml1|binary1|json|swift|objc}} {{file.plist}} -o {{new_file.plist}}`

Reviewer: ```suggestion
`plutil -convert {{xml1|binary1|json|swift|objc}} {{path/to/file.plist}} -o {{path/to/new_file.plist}}`
```

> - Convert a single plist file to a different format, writing to stdout:

Reviewer: ```suggestion
- Convert a plist file to a different format, writing to `stdout`:
```

> `plutil -convert {{xml1|binary1|json|swift|objc}} {{file.plist}} -o -`

Reviewer: ```suggestion
`plutil -convert {{xml1|binary1|json|swift|objc}} {{path/to/file.plist}} -o -`
```
